include/rocksdb/memtablerep.h

```diff
@@ -81,7 +81,7 @@ class MemTableRep {
   // single buffer and pass that in as the parameter to Insert).
   // REQUIRES: nothing that compares equal to key is currently in the
   // collection, and no concurrent modifications to the table in progress
-  virtual void Insert(KeyHandle handle) = 0;
+  virtual bool Insert(KeyHandle handle) = 0;
```

Comment:
* Comment what's return means
* InsertWithHint and InsertConcurrent should also return duplicate key error.
* Shall we handle duplicate keys for other memtable types if that's not too difficult to do? Or alternatively, shall we add a method (e.g. MemTableRep::CanHandleDuplicatedKey() ?) to denote the behavior difference?